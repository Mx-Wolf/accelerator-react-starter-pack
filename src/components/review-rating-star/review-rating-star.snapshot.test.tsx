import renderer from 'react-test-renderer';
import ReviewRatingStar from './review-rating-star';

const fakeStar = '2';
const fakeLabel = 'test';
const fakeOnChange = jest.fn();
const fakeUserRating = 2;

describe('Component: ReviewRatingStar', () => {
  it('should render correctly', () => {
    const component = renderer.create(<ReviewRatingStar star={fakeStar} userRating={fakeUserRating} onChange={fakeOnChange} label={fakeLabel}/>);
    expect(component).toMatchSnapshot();
  });
});
