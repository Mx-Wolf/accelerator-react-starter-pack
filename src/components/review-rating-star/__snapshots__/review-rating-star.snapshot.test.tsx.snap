// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: ReviewRatingStar should render correctly 1`] = `
Array [
  <input
    className="visually-hidden"
    id="star-2"
    name="rate"
    onChange={[Function]}
    type="radio"
    value="2"
  />,
  <label
    className="rate__label"
    htmlFor="star-2"
    title="test"
  />,
]
`;
